ceph/rados/core_workflows.py

```diff
@@ -271,6 +271,97 @@ def get_pool_property(self, pool, props):
         prop_details = json.loads(out)
         return prop_details
 
+    def host_maintenance_enter(self, hostname: str, retry: int = 10) -> bool:
+        """
+        Adds the specified host into maintenance mode
+        Args:
+            hostname: name of the host which needs to be added into maintenance mode
+            retry: Retry the command to add host into maintainence mode
```

Comment:
[minor] Please update the text to _`max number of retries to put host into maintenance mode`_